JSDemos/Demos/Charts/Spline/description.md

```diff
@@ -1 +1,7 @@
-Spline series types, like line series types, help visualize data trends over specified intervals. However, splines help &laquo;smooth&raquo; individual lines that connect data points. In&nbsp;this example, you can switch between the _spline_, _stacked spline_ and _full-stacked spline_ series types. _Stacked spline_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked spline_ series help compare the percentage value of&nbsp;multiple spline series for each argument.
\ No newline at end of file
+Spline charts, like [line charts](/Demos/WidgetsGallery/Demo/Charts/Spline/jQuery/Light/), allow you to visualize data and display them as lines with points placed over specified intervals. However, spline charts smooth out individual lines that connect data points. 
+
+In this example, you can switch through the Spline, Stacked Spline and Full-Stacked Spline charts. A Stacked Spline chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Spline chart helps compare the percentage value of multiple line series for each argument. You can use the drop-down menu under the chart to switch between spline chart types mentioned above.
+
+## Configure Series Type
+
+The series type defines the appearance of your chart. To display a Spline, Stacked Spline, or Full-Stacked Spline series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types. 
```

Comment:
We need to say clearly that for information about configuration the reader should go to the Line demo, with a note that for spline types the `type` property should be set to different values.